test/abstract_syntax_tree.coffee

```diff
@@ -1006,80 +1006,211 @@ test "AST as expected for Index node", ->
 #     body:
 #       type: 'Block'
 
-# test "AST as expected for Op node", ->
-#   testExpression '1 <= 2',
-#     type: 'Op'
-#     operator: '<='
-#     originalOperator: '<='
-#     flip: no
-#     first:
-#       value: '1'
-#     second:
-#       value: '2'
+test "AST as expected for Op node", ->
+  testExpression '1 <= 2',
+    type: 'BinaryExpression'
+    operator: '<='
+    left:
+      type: 'NumericLiteral'
+      value: 1
+    right:
+      type: 'NumericLiteral'
+      value: 2
 
-#   testExpression '1 is 2',
-#     type: 'Op'
-#     operator: '==='
-#     originalOperator: 'is'
-#     flip: no
+  testExpression '1 is 2',
+    type: 'BinaryExpression'
+    operator: 'is'
+    left:
+      type: 'NumericLiteral'
+      value: 1
+    right:
+      type: 'NumericLiteral'
+      value: 2
 
-#   testExpression '1 // 2',
-#     type: 'Op'
-#     operator: '//'
-#     originalOperator: '//'
-#     flip: no
+  testExpression '1 // 2',
+    type: 'BinaryExpression'
+    operator: '//'
+    left:
+      type: 'NumericLiteral'
+      value: 1
+    right:
+      type: 'NumericLiteral'
+      value: 2
 
-#   testExpression '1 << 2',
-#     type: 'Op'
-#     operator: '<<'
-#     originalOperator: '<<'
-#     flip: no
+  testExpression '1 << 2',
+    type: 'BinaryExpression'
+    operator: '<<'
+    left:
+      type: 'NumericLiteral'
+      value: 1
+    right:
+      type: 'NumericLiteral'
+      value: 2
```

Comment:
It’s a bad idea to use 1 (or 0) as values in tests, because they can also equal true or false. Just pick any other numbers: 3, 42, whatever.